src/project-2.js

```diff
@@ -1,44 +1,73 @@
 // Do not change any of the function names
 
 const getBiggest = (x, y) => {
+  if (x > y) {
+    return x;
+  }
+  return y;
+
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
 };
 
 const greeting = (language) => {
+  const translations = {
+    German: 'Guten Tag!',
+    Spanish: 'Hola!',
+    Chinese: 'Ni Hao!',
+  };
+  return translations[language] || 'Hello!';
```

Comment:
This is a creative approach.  You could also do something along the lines of setting a default parameter: `(language = 'English') => {`.